xclim/indices/generic.py

```diff
@@ -1431,24 +1493,24 @@ def get_zones(
     Parameters
     ----------
     da : xr.DataArray
-        Input data
-    zone_min : Quantity | None
-        Left boundary of the first zone
-    zone_max : Quantity | None
-        Right boundary of the last zone
-    zone_step: Quantity | None
-        Size of zones
-    bins : xr.DataArray | list[Quantity] | None
-        Zones to be used, either as a DataArray with appropriate units or a list of Quantity
-    exclude_boundary_zones : Bool
+        Input data.
+    zone_min : Quantity, optional
+        Left boundary of the first zone.
+    zone_max : Quantity, optional
+        Right boundary of the last zone.
+    zone_step : Quantity, optional
+        Size of zones.
+    bins : xr.DataArray or list of Quantity], optional
```

Comment:
hum ... so leaving out pipe operators is intended?  or could we write: 

```suggestion
    bins : xr.DataArray | list[Quantity], optional
```